### PR TITLE
Configure log level

### PR DESCRIPTION
**Pull request by Rados13**
## Acknowledging the stipulations set forth:
 - [x] I hereby confirm that a Pull Request involving updates to the Software Development Kit (SDK) has been smoothly merged, currently awaits processing, or is otherwise deemed unnecessary in this context.
 - [x] I also affirm that another Pull Request, specifically addressing updates to the documentation body (commonly referred to as 'docs'), has either been successfully incorporated, is in the process of review, or is considered superfluous under the prevailing circumstances.
 
[ PR to FJ docs](https://github.com/fishjam-dev/fishjam-docs/pull/111)

### CODE REVIEW

File: lib/jellyfish/component/hls/manager.ex [hunk 1]
```diff
@@ -75,7 +75,12 @@ defmodule Fishjam.Component.HLS.Manager do
       end)
 
     broadcast_notification(result, room_id)
-    Logger.info("Finished uploading to s3 with result: #{result}, room: #{room_id}")
+
+    if is_tuple(result) do
+      Logger.info("Finished uploading to s3 with result: #{inspect(result)}, room: #{room_id}")
```
Comment (kamciokodzi) [inline lib/jellyfish/component/hls/manager.ex:80]: I think this if is not necessary, you could just use this one in all cases.

File: lib/jellyfish/config_reader.ex [hunk 2]
```diff
@@ -246,6 +246,23 @@ defmodule Fishjam.ConfigReader do
     get_env("FJ_GIT_COMMIT", "dev")
   end
 
+  def read_logger_level() do
+    log_level = System.get_env("FJ_LOG_LEVEL", "info")
```
Comment (sgfn) [inline lib/jellyfish/config_reader.ex:250]: I wonder whether we should handle `JF_LOG_LEVEL` as well. I know this env var didn't exist prior to the rebranding, but I think it's better to be consistent in this matter. In a few versions we'll drop the `JF_` support anyway

```suggestion
    log_level = get_env("FJ_LOG_LEVEL", "info")
```

**Review by sgfn — APPROVED** [review]

File: config/dev.exs [hunk 3]
```diff
@@ -26,5 +26,3 @@ config :phoenix, :stacktrace_depth, 20
 
 # Initialize plugs at runtime for faster development compilation
 config :phoenix, :plug_init_mode, :runtime
-
-config :logger, level: :info
```
Comment (kamciokodzi) [inline config/dev.exs:30]: Why? Do we want to include `debug` logs by default as well?

File: lib/jellyfish/config_reader.ex [hunk 4]
```diff
@@ -246,6 +246,23 @@ defmodule Fishjam.ConfigReader do
...
+
+    log_levels_string = ["info", "debug", "warning", "error"]
+
+    if log_level in log_levels_string do
+      String.to_atom(log_level)
```
Comment (kamciokodzi) [inline lib/jellyfish/config_reader.ex:255]: I would consider `String.to_existing_atom/1`

File: config/dev.exs (hunk 3 above)
Comment (mickel8) [reply config/dev.exs:30]: Now, log level is configured in runtime.exs where info is the default value

**Review by mickel8 — APPROVED** [review]
Can we merge this after https://github.com/fishjam-dev/fishjam/pull/199?

File: config/dev.exs (hunk 3 above)
Comment (kamciokodzi) [reply config/dev.exs:30]: Ah yes my mistake, thought it was in prod not runtime 😅

**Review by kamciokodzi — APPROVED** [review]

**Pull request merged**